### PR TITLE
Updated hcc-voms.unl.edu (SOFTWARE-3990)

### PR DESCRIPTION
**Pull request by hzhu16**

### CODE REVIEW

**Comment by edquist** [comment]
@hzhu16, @brianhlin, what are the intended DNs/CAs for hcc-voms.unl.edu / hcc-voms2.unl.edu ?

Looks like something bad happened with a merge when a revert was maybe intended.

It's not clear from the commit messages or the PR title/description what the intended results are.

File: vomses [hunk 1]
```diff
@@ -16,8 +16,8 @@
 "Gluex" "gluex.phys.uconn.edu" "15014" "/DC=org/DC=incommon/C=US/ST=CT/L=Storrs/O=University of Connecticut/OU=Academic IT/CN=gluex.phys.uconn.edu" "Gluex"
 "Gluex" "gryphn.phys.uconn.edu" "15014" "/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=voms/gryphn.phys.uconn.edu" "Gluex"
 "GridUNESP" "voms.grid.unesp.br" "15000" "/C=BR/O=ANSP/OU=ANSPGrid CA/OU=Services/CN=voms.grid.unesp.br" "GridUNESP"
-"hcc" "hcc-voms.unl.edu" "15000" "/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=hcc-voms.unl.edu" "hcc"
-"hcc" "hcc-voms2.unl.edu" "15000" "/DC=org/DC=incommon/C=US/ST=Nebraska/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu" "hcc"
+"hcc" "hcc-voms.unl.edu" "15000" ""/DC=org/DC=incommon/C=US/ST=Nebraska/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms.unl.edu" "hcc"
```
Comment (brianhlin) [inline vomses]:
```suggestion
"hcc" "hcc-voms.unl.edu" "15000" "/DC=org/DC=incommon/C=US/ST=Nebraska/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms.unl.edu" "hcc"
```

**Comment by brianhlin** [comment]
@edquist the subject DNs in vomses look right to me. The issuer DNs for both of them should be the InCommong IGTF CA

**Comment by edquist** [comment]
is it really correct that `hcc-voms.unl.edu.lsc`'s DN has `/CN=hcc-voms2.unl.edu` ?

**Comment by brianhlin** [comment]
It looks like it has `/CN=hcc-voms.unl.edu` to me: https://github.com/opensciencegrid/osg-vo-config/blob/340be4249f14ac05da71c5b8c3037e681c50a600/vomsdir/hcc/hcc-voms.unl.edu.lsc#L1

**Comment by edquist** [comment]
Nevermind i was looking at something wrong.

**Comment by edquist** [comment]
Ok, just to confirm then, we want:

`ST=Nebraska` for `hcc-voms.unl.edu` but `ST=NE` for `hcc-voms2.unl.edu` ?

**Comment by edquist** [comment]
Brian says yes, so we'll go ahead with this.

**Pull request merged**